### ISSUE TITLE
Wrong path on the example How to use

### ISSUE THREAD

**Issue by razier** [roelandmoors/authjs-adapter-sql#6]
Hi, First of all thanks for the adapter (I didn't want to use Prisma for such a small scale hobby project)

However based on the example in README.md

```
import buildPlanetScaleHelpers from "authjs-adapter-mysql/dist/planetscale";
```
I am getting the error
**Module not found: Package path ./dist/planetscale is not exported from package** 

Looking at the package.json the example should instead be
```
import buildPlanetScaleHelpers from "authjs-adapter-mysql/planetscale";
```

**Comment by roelandmoors** [comment]
Hi, I renamed the package to [authjs-adapter-sql](https://www.npmjs.com/package/authjs-adapter-sql) because I also want to support `vercel/postgres` at some point.
Could you try the latest version 0.1.1? 
I will keep supporting mysql/planetscale. (I use the same sql)

This should work:
`import buildPlanetScaleHelpers from "authjs-adapter-mysql/dist/planetscale";`

I'm not sure why you get an error. 
I did some tests with adding exports in package.json, but I removed them because it didn't seem to work.
(I need to investigate that a bit more)

You can also copy [that helper code](https://github.com/roelandmoors/authjs-adapter-sql/blob/main/src/planetscale.ts) to your own project if you want.

**Comment by razier** [comment]
Updated to 0.1.1 with this error

![image](https://user-images.githubusercontent.com/1881987/235548810-21095cbe-8eb6-4344-80e8-faa03c216f8f.png)

It only works after I modify and added following to the authjs-adapter-sql package.json

![image](https://user-images.githubusercontent.com/1881987/235549057-a71f46c9-21e2-402a-bf41-b31fd4486600.png)

**Comment by roelandmoors** [comment]
Are you using Nextjs with javascript (not typescript)?

I'm testing with typescript and that seems to behave a bit different.
I added the exports back in commit 441ebdee6eb51603aed8e9f0554fdda5626f255d and released version 0.1.2
Could you try again?

**Comment by razier** [comment]
Ah yes I am working on nextjs with javascript.

I had tested 0.1.2 and follow the import based on the latest readme.md, everything works accordingly based on the documentation now.

Thank you once again for the adapter and for updating!
